_src/static/sass/components/_colors.scss

```diff
@@ -232,18 +232,6 @@
         background: palette(accent, x-dark);
     }
 
-    &.mid-sat {
```

Comment:
In our particular PL case, the grays that had the saturation was our brand accent. So, maybe we need to rename that gray to be neutral or something?